### DISCUSSION TITLE
How to display the cover image on /post

### DISCUSSION THREAD

**Discussion by pedrobrantes** [adityatelange/hugo-PaperMod#1447] [Q&A] [answered]
## An example of what I want:
![image](https://github.com/adityatelange/hugo-PaperMod/assets/58346706/650ab26c-6900-4028-a2cc-26b9b326987e)

### I have already set the cover parameters
![image](https://github.com/adityatelange/hugo-PaperMod/assets/58346706/e13118aa-8b3f-403a-b1db-7144916f52ca)

### but the cover is only displayed in the post
![image](https://github.com/adityatelange/hugo-PaperMod/assets/58346706/1981b706-cb01-441b-bf34-b7695459e851)

**Comment by vand3rlinden** [comment]
You can set the following syntaxes in your ```.md``` post.

```
cover: 
  image: /images/yourimage.png
```

- [Example MarkDown](https://raw.githubusercontent.com/vand3rlinden/vand3rlinden.github.io/main/content/post/spf-dkim-dmarc-explanation.md)
- [Example on PaperMod](https://vand3rlinden.com/post/spf-dkim-dmarc-explanation/)

**Edit:** Personally, I don't think you want a cover img on your post page, it might take up too much space. See also this [video](https://youtu.be/hjD9jTi_DQ4?list=PLeiDFxcsdhUrzkK5Jg9IZyiTsIMvXxKZP&t=2703)

**Reply by pedrobrantes** [reply]
What I want is for this cover image to also appear on /post.

**Reply by pedrobrantes** [reply, marked as answer]
I succeeded, some parameters were missing in ``cover``.
```yaml
cover:
        hidden: false
        hiddenInList: false
        hiddenInSingle: false
```

**Reply by emranalus** [reply]
> I succeeded, some parameters were missing in `cover`.
> 
> ```yaml
> cover:
>         hidden: false
>         hiddenInList: false
>         hiddenInSingle: false
> ```

I am trying to solve this for hours man thank you so much, btw do you happen to know how to add favicon to my site bcs whatever I try papermod just doesn't see my favicon.ico which is in static.

**Comment by jakwoda** [comment]
Hi, 
I also have similar problem. Cannot add favicon and the cover images on the list.html page?

Can anyone point me into what could be wrong or how to check it? It's my first encounter with Hugo and Notion as CMS

https://github.com/jakwoda/notionTest
